docs/src/writing-tests-js-python.md

```diff
@@ -128,8 +168,20 @@ Playwright also includes async matchers that will wait until the expected condit
 await expect(page).toHaveTitle(/Playwright/);
 ```
 
+## Assertions
+* langs: python
+
+Playwright includes [assertions](./test-assertions.md) that will wait until the expected condition is met. Using these assertions allows making the tests non-flaky and resilient. For example, this code will wait until the page gets the title containing "Playwright":
 
+```python
+import re
+from playwright.sync_api import expect
 
+expect(page).to_have_title(re.compile("Playwright"))
+```
+######
+* langs: js, python
```

Comment:
```suggestion
```

I think you can remove this here right, since the whole docs is only for py/nodejs, so the check seems not relevant anymore.

Reply:
well its there for our benefit. we need the empty heading otherwise only prints in one language. this section is a shared section but is part of the assertions so it doesn't have a heading. if I leave the markdown and remove the langs part it looks too easy to just remove the empty heading and then it wont work across langs. so really this is our only way of splitting part of a section up just like when we add images, fake headings to trick the generator

Reply:
sg